### ISSUE TITLE
Please add “quick start” w/install+usage to README

### ISSUE THREAD

**Issue by bionicles** [scalegenius/pg_bitemporal#33]
To know how to begin, as a developer, I like to follow some install + usage guide in the readme rather than hunt around. Could we add a couple code blocks to the readme for how to install this and do basic CRUD / use cases?

 This project would be more useful with a README (rather than a folder of docs which is hard to parse, just show us how to begin and use it right away) 

A full API doc would be handy but is more work with diminished return. Just install and usage examples would be 80:20

Remember README is like the front page first impression of the project. If the README looks legit, so does the project, and by extension, the contributors. Thanks for making this

**Comment by hettie-d** [comment]
Hi, thank you so much for your feedback! My apologies for not responding earlier - I am on medical leave. We've already received similar feedback from other pg_bitemporal users, and we understand the importance of the "Quick Start" section. We will work on adding it to the library and will inform you when we have it. Thank you for trying pg_bitemporal!

**Comment by aakarshg** [comment]
Hey @hettie-d Is there a quickstart to install the extension ? I stumbled upon this and currently trying to figure out how can I possibly install this extension.. So any guide would be nice

**Comment by hettie-d** [comment]
The short answer: clone the repo, start the psql command line tool from the pg_bitemporal/sql directory and run _load_all.sql file. Please let me know whether this helps!
